proto/osmosis/streamswap/v1/tx.proto

```diff
@@ -0,0 +1,138 @@
+syntax = "proto3";
+package osmosis.streamswap.v1;
+
+import "gogoproto/gogo.proto";
+import "google/protobuf/duration.proto";
+import "google/protobuf/timestamp.proto";
+import "google/protobuf/empty.proto";
+
+import "cosmos/base/v1beta1/coin.proto";
+import "cosmos_proto/cosmos.proto";
+
+option go_package = "github.com/osmosis-labs/osmosis/v10/x/streamswap/types";
+option (gogoproto.goproto_getters_all) = false;
+
+service Msg {
+  // CreateSale creates new token sale. Anyone can create a new sale.
+  // params.SaleBond OSMO will be charged as a bond (returned in FinalizeSale)
+  // to avoid spams.
+  // The sale follows the streamswap functionality explained in the
+  // x/launchapd/spec
```

Comment:
Is there a draft of the spec available?

Reply:
no, it's other task in the linked issue. Spec will use proto from this PR.